README.md

```
@@ -39,6 +39,10 @@ It is part of the [_Munich Quantum Toolkit (MQT)_](https://mqt.readthedocs.io).
 > Usage for _Lattice Surgery Compilation Beyond the Surface Code_ as well as _Exploiting Movable Logical Qubits for Lattice Surgery Compilation_ is described in [`docs/cococo.md`](https://github.com/munich-quantum-toolkit/qecc/blob/cococo/docs/cococo.md) in the `cococo` branch.
 > The code quality in the branch is actively being improved.
 
+> [!NOTE]
+> Usage for _Synthesis of Fault-tolerant State Preparation Circuits using Steane-type Error Detection_ can be found in the branch ['ft-stateprep-qubit-perm'](https://github.com/munich-quantum-toolkit/qecc/tree/ft-stateprep-qubit-perm).
+> The documentation is a work in progress and this branch undergoes some final improvements before being merged into `main`.
+
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)